doc/api.md

```diff
@@ -393,6 +393,27 @@ Object of:
 
 ---
 
+### media/metadata/create (POST,PUT; AUTH)
+
+#### Parameters
+
+Parameters are sent as JSON object:
+
+```
+{
+	"media_id":"1234",
+	"alt_text": {
+		"text":"Here comes the description"
```

Comment:
Is it a standard data structure?

Reply:
What do you mean with this? In fact I did the coding in reference to this documentation: https://developer.twitter.com/en/docs/media/upload-media/api-reference/post-media-metadata-create.html

Reply:
That was my question, thanks.